multinet/util.py

```diff
@@ -40,3 +43,10 @@ def decode_data(input: bytes) -> str:
         raise DecodeFailed(str(e))
 
     return body
+
+
+def test_data_path(file_name: str) -> str:
+    """Load data from the test directory."""
+    file_path = os.path.join(TEST_DATA_DIR, file_name)
+    print(file_path)
+    return file_path
```

Comment:
I think this function should go directly in the file where it's used.

Reply:
My plan is to have this be usable by all the tests to reduce code clutter. Does that sound good or should I move it and duplicate it to the other tests in another PR?

Reply:
No you're right, it should stay here. Just change the name of the function so pytest doesn't get confused by it. Maybe just `data_path()`?

Reply:
Fixed in e7bca47